packages/apps/docs/src/components/Specs/styles.css.ts

```diff
@@ -0,0 +1,68 @@
+import { vars } from '@kadena/react-ui/theme';
+
+import { globalStyle, style } from '@vanilla-extract/css';
+
+export const specsWrapper = style({});
+
+globalStyle(`${specsWrapper} .menu-content li span`, {
+  color: vars.colors.$foreground,
+});
+
+globalStyle(`${specsWrapper} .menu-content li polygon`, {
+  fill: vars.colors.$foreground,
+});
+
+globalStyle(`${specsWrapper} .api-content`, {
+  width: '100%',
+  color: vars.colors.$foreground,
+});
+
+globalStyle(`${specsWrapper} .api-content polygon`, {
+  fill: vars.colors.$foreground,
+});
+
+globalStyle(`${specsWrapper} .api-content h1`, {
+  lineHeight: '100%',
+  color: vars.colors.$foreground,
+});
+
+globalStyle(`${specsWrapper} .api-content h2`, {
+  lineHeight: '100%',
+  color: vars.colors.$foreground,
```

Comment:
`Specs` component used for displaying api specification document.

the classes like `.api-content`, menu-content` and all the related styles are coming from the different doc files but as part of docs need to add some extra styles.  

I was checking how to target and style based on nested classes `.specs-wrapper .api-content`. I found only option of doing it with `globalStyle` but if there is an any other better options in VE please let me know.

Reply:
Yeah, generally we want to apply the styles directly to the element rather than from the parent (so avoid using globalStyle), but if it's necessary then this is fine 👍